src/dpq2/conv/from_d_types.d

```diff
@@ -78,11 +78,22 @@ Value toValue(T)(T v)
 if (is(Unqual!T == TimeStampWithoutTZ))
 {
     long j = v.dateTime.julianDay - POSTGRES_EPOCH_DATE.julianDay; // FIXME: use POSTGRES_EPOCH_JDATE here
+    if (v.hour < 12) j += 1; // fix before/after noon values difference
```

Comment:
This isn't same thing as "modified Julian day"?
https://dlang.org/phobos/std_datetime_date.html#.DateTime.modJulianDay
https://en.wikipedia.org/wiki/Julian_day : Modified Julian Date (MJD)